package.json

```diff
@@ -29,6 +29,7 @@
         "angular-fittext": "2.1.1",
         "angular-translate-interpolation-messageformat": "2.18.1",
         "angular2-moment": "1.9.0",
+        "angulartics2": "^7.4.1",
```

Comment:
please avoid the ^ because it can lead to broken builds due to changes in new versions (we had this several times unfortunately)